Add Alias of mulitpath-tools because that is what we named our init service
--- a/multipathd/multipathd.service
+++ b/multipathd/multipathd.service
@@ -20,3 +20,4 @@
 [Install]
 WantedBy=sysinit.target
 Also=multipathd.socket
+Alias=multipath-tools.service
